src/frontend/DevolioFront/src/app/portfolio/components/portfolio/portfolio.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-portfolio',
+  templateUrl: './portfolio.component.html',
+  styleUrls: ['./portfolio.component.scss']
+})
+export class PortfolioComponent implements OnInit {
```

Comment:
What's this component for? Is it for a future usecase or is it going to be implemented in this PR?